### DISCUSSION TITLE
What are the best configurations for expected 500-1500 users using hubs and spoke ?

### DISCUSSION THREAD

**Discussion by RathodDeven** [mozilla/hubs#5739] [Hubs Cloud Questions]
Hey, 
We are planning for user launch by this month. Super excited about it! We have been using and building upon custom hubs cloud and added so many unique features. One of them is NFT-authenticated access to the product.

And we are expecting 500 - 1500 users to have access to the full product (hubs/spoke). 
What would be the best server configuration we can use and move forward with? Any suggestions are appreciated.

**Comment by pattersonbl2** [comment]
Currently, Hubs is unable to support this configuration.   We can do up to 50 people a room now, there can be possible two rooms like this but it may cause performance issues.
